src/main/java/service/ReaderData.java

```diff
@@ -0,0 +1,8 @@
+package service;
+
+import java.nio.file.Path;
+import java.util.List;
+
+public interface ReaderData {
```

Comment:
Don't forget about naming convention. What it is? Simple class? Service? Dao? 
Fix it for all interfaces and their implementations